src/system.jl

```diff
@@ -387,11 +460,68 @@ Base.@kwdef mutable struct SystemDA <: System
 
     # Virtuals/PSD time series
     "Increment bids time series data. `KeyedArray` where the axis keys are `bid ids x datetimes`"
-    increments::KeyedArray{Vector{Tuple{Float64, Float64}}, 2}
+    increments::KeyedArray{PriceSensitiveBid, 2}
     "Decrement bids time series data. `KeyedArray` where the axis keys are `bid ids x datetimes`"
-    decrements::KeyedArray{Vector{Tuple{Float64, Float64}}, 2}
+    decrements::KeyedArray{PriceSensitiveBid, 2}
     "Price sensitive load bids time series data. `KeyedArray` where the axis keys are `bid ids x datetimes`"
-    price_sensitive_loads::KeyedArray{Vector{Tuple{Float64, Float64}}, 2}
+    price_sensitive_loads::KeyedArray{PriceSensitiveBid, 2}
+end
+
+"""
+    $TYPEDEF
+
+Subtype of a `System` for modelling a purely financial style of day-ahead market clearing.
+Key differences to the physical day-ahead market clearing approach are that _all_ offers and
+bids are price sensitive and virtual participants are not distinguished from physical
+participants.
+
+Fields:
+$TYPEDFIELDS
+"""
+Base.@kwdef mutable struct FinancialSystemDA <: System
```

Comment:
I'm not sure about this name since it clashes with another widely-used susbystem in our codebase that we are probably going to have to deal with eventually. Is there another appropriate name?